src/migration-wizard/steps/connect/sources-table/empty-state/ProxyFields.tsx

```
@@ -164,7 +160,7 @@ const ProxyFields: React.FC = () => {
           id="enable-proxy"
           label="Enable proxy"
           isChecked={enableProxy}
-          onChange={(_event, value) => toggleEnableProxy(value)}
+          onChange={(_event,value) => toggleEnableProxy(value)}
```

ESLint

[error] Insert `·`

(prettier/prettier)